### ISSUE TITLE
Integrate display of media file hash from IA

### ISSUE THREAD

**Issue by n8fr8** [OpenArchive/Save-app-android#33]
IA provides md5 / sha1 hashes after publish. We should grab those, compare locally, and store for future display. Also sha256.

**Comment by elric-wamugu** [comment]
@n8fr8 @foundscapes this seems to have already been implemented in the recent OA versions, yes?

**Comment by foundscapes** [comment]
@hiromipaw let's ensure this is working and then we can close

**Comment by hiromipaw** [comment]
@purvi-ranawat can we test this?

**Comment by purvi-ranawat** [comment]
@hiromipaw : The issue is still not resolved as per below communication on 29th Sept 2023 : 

![image](https://github.com/OpenArchive/Save-app-android/assets/8190141/c7d0aaa9-2084-4c25-899c-0dc83272757d)

**Comment by foundscapes** [comment]
@tladesignz please determine if this can be fixed or if it's a deprecated issue that we no longer need to be concerned with.

**Comment by tladesignz** [comment]
That is a business decision. 

My 2 cents on this:
- I don't see the point what this should be for, but I'm no subject matter expert, and I'm certainly no big Internet Archive user.
-I also don't see any UI coming along to show anything like this.

**Comment by foundscapes** [comment]
Yeah, will add to future coolness bc issues is now 7 years old, so would need more research than we have time for rn

**Comment by upulw2020** [comment]
@foundscapes @purvi-ranawat IA cannot be logged into since the hack. They are maintaining a readonly service atm. We should probably disable it or hide it as an option.
